TFT/src/User/Menu/Popup.c

```diff
@@ -124,27 +124,14 @@ void menuDialog(void)
   }
 }
 
-void popup_strcpy(uint8_t *dst, uint8_t *src, uint16_t size)
```

Comment:
I'm not sure if `memcpy` is safe when `const char *src` is `NULL`, so I will refactor it and submit commit for review later